app/views/teams/show.html.slim

```diff
@@ -86,12 +86,11 @@ nav.page
     h3 Supervisors
     p = @team.supervisors.any? ? link_to_team_members(@team, :supervisor) : content_tag(:em, 'Will be added by RGSoC Organizers')
 
-- if @team.members.any? { |user| user.conferences.in_current_season.any? }
+- if @team.conferences.in_current_season.any?
```

Comment:
Same as for the orga teams view - I think there's something wrong with the list layout.